src/app/(guest)/scoreboard/[id]/page.tsx

```diff
@@ -20,42 +23,44 @@ const ScoreboardPage: React.FC<ScoreboardPageProps> = async ({ params }) => {
     return res.json();
   }
 
+  const example: Score = {
+    user_id: 1000,
+    score: 51.56,
+    user_name: 'Alguém (You)',
+    subject_id: 1,
+    subject: 'algan',
+    avatar: '1',
+    exams: 5
+  };
+
   return (
     <section className="min-h-[90vh] flex flex-col items-center my-16">
       <p className="text-xl font-bold uppercase text-center px-4">
         Scoreboard de <span className="text-primary">{subjectName}</span>
       </p>
 
-      <section className="mt-5 w-full md:px-16 grid place-items-center">
+      <section className="my-5 w-full grid place-items-center">
         {scoreboard.scores.length === 0 ? (
           <p className="text-center">Sem nenhum utilizador registado</p>
         ) : (
-          <table className="w-1/2 text-sm text-center">
-            <thead className="text-xs text-white uppercase bg-primary">
-              <tr>
-                <th scope="col" className="px-6 py-3">
-                  Posição
-                </th>
-                <th scope="col" className="px-6 py-3">
-                  Nome
-                </th>
-                <th scope="col" className="px-6 py-3">
-                  Pontuação
-                </th>
-              </tr>
-            </thead>
-            <tbody>
-              {scoreboard.scores.map((line, key) => (
-                <tr className="bg-white border-b" key={line.subject_id}>
-                  <th scope="row" className="px-6 py-4 font-medium text-primary whitespace-nowrap">
-                    {key + 1}
-                  </th>
-                  <td className="px-6 py-4">{line.user_name}</td>
-                  <td className="px-6 py-4">{line.score}</td>
-                </tr>
-              ))}
-            </tbody>
-          </table>
+          <>
+            {/* Year Selector, future implementation */}
```

Comment:
Nice one, needs implementation on backend and needs checking in terms of infrastructure, but great idea